cmd/start.go

```diff
@@ -140,6 +140,11 @@ func Start(c *cli.Context) error {
 		return err
 	}
 
+	err = config.Extensions.TBTC.Bitcoin.Validate()
+	if err != nil {
+		logger.Warnf("invalid bitcoin configuration for tbtc extension: [%v]", err)
+	}
```

Comment:
What if we cover two possible situations differently:
1. `Extensions.TBTC.Bitcoin` configuration is not provided --> we log a warning and the client starts (backward compatibility).
2. `Extensions.TBTC.Bitcoin` configuration is provided but is invalid --> we return an error so the client won't start

For the second case, we could also just log an error and let the client start, the client will possibly fail a recovery if liquidation happens in the future.

I'd like to ask @Shadowfiend for his opinion too.

Reply:
Okay - implemented this idea in 4bdbc63. I'm not in love with *how* it's done, so if you have a more elegant way I'm all ears!

Reply:
What if we use `(bitcoin.Config{}) == config.Extensions.TBTC.Bitcoin`?
```go
err := config.Extensions.TBTC.Bitcoin.Validate()
if err != nil {
	if (bitcoin.Config{}) == config.Extensions.TBTC.Bitcoin {
		logger.Warnf("missing bitcoin configuration for tbtc extension: [%v]", err)
	} else {
		return err
	}
}
```